### PR TITLE
bsnes-mt: Add version 1.3.4

### PR DESCRIPTION
**Pull request by RavenMacDaddy**
bsnes-mt is a fork of bsnes that adds video features like pixel-perfect, integer-ratio scaling, as well as the aspect ratio that corresponds to that of the SNES console.

- [X] I have read the [Contributing Guide](https://github.com/Calinou/scoop-games/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by RavenMacDaddy** [comment]
Not sure what's going wrong.

I can reinstall the app with this manifest however many times I want - but please, if you'd like me to fix something, just ask.

**Comment by Calinou** [comment]
Regarding the license, I'd just list the same license as bsnes.

**Comment by RavenMacDaddy** [comment]
> Regarding the license, I'd just list the same license as bsnes.

Let's try it.

**Comment by sitiom** [comment]
> Not sure what's going wrong.

About the failure, the `WindowsPowerShell` job seems to be also failing on the official repos, so just ignore it.

**Comment by RavenMacDaddy** [comment]
Reopening to retry checks.

**Comment by RavenMacDaddy** [comment]
```
CommandNotFoundException: The term 'warn' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
      at parse_json, D:\a\scoop-games\scoop-games\scoop_core\lib\manifest.ps1: line 10
      at <ScriptBlock>, D:\a\scoop-games\scoop-games\scoop_core\test\Import-Bucket-Tests.ps1: line 118
```

The only thing I can find when I search for _scoop core_ is the [Ash258 fork](https://github.com/Ash258/Scoop-Core), which hasn't been updated for like two months, even on the dev branch.

**Comment by RavenMacDaddy** [comment]
@BlackPowerade, if you wouldn't mind, help me understand what's failing – I'm out of ideas by this point.

File: bucket/bsnes-mt.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        ]
+    ],
+    "persist": [
+        "Database",
+        "Firmware",
+        "Shaders",
+        "bsnes-mt-settings.bml"
+    ],
+    "checkver": {
+        "github": "https://github.com/Marat-Tanalin/bsnes-mt",
```
Comment (BlackPowerade) [inline bucket/bsnes-mt.json:35]: trailing comma - needs to be removed

**Review by BlackPowerade — COMMENTED** [review]
The trailing comma is likely what is making it fail.    
That being said I don't know why PS5 passes but PS7 fails. Maybe warning behavior changed in 7? ¯\_(ツ)_/¯    
I'll look into it a bit more.

**Comment by RavenMacDaddy** [comment]
Of course, I missed that…because that won't stop the app from installing.

Thank you, @BlackPowerade – as always, the hardest problems are also the most obvious.

**Comment by RavenMacDaddy** [comment]
Finally, it should be done.

@Calinou, would you mind taking a last look if everything looks ok before merging?

**Comment by BlackPowerade** [comment]
It's no problem raven. I make these kind of mistakes all the time.     
What are you using to write manifests? IDE should have picked this up.

**Comment by RavenMacDaddy** [comment]
> It's no problem raven. I make these kind of mistakes all the time. What are you using to write manifests? IDE should have picked this up.

Honestly, I think I mostly use the GitHub view, as that often picks up obvious mistakes when I've created manifests so far – but I'll gladly take suggestions.

One app that I do use in some cases is **Notepad++** – maybe there's a plugin for that? I already use it to format for JSON properly if there are things like white space errors, etc.

**Comment by BlackPowerade** [comment]
I use VS code, it works as a swiss army knife for all kinds of development work.     

Also I think I found the issue with PS failing. https://github.com/Ash258/Scoop-Core/blob/main/lib/core.ps1#L984    
Something is going on in the CI where this function is being called either improperly or the function is not getting declared before it is being called. That's all I know right now.

**Comment by RavenMacDaddy** [comment]
> I use VS code, it works as a swiss army knife for all kinds of development work.

This will be the last off-topic, but yes, if I'd used VSC to begin with, I would've gotten it from the start; 

image

**Comment by sitiom** [comment]
> Also I think I found the issue with PS failing. [Ash258/Scoop-Core@`main`/lib/core.ps1#L984](https://github.com/Ash258/Scoop-Core/blob/main/lib/core.ps1?rgh-link-date=2022-09-18T19%3A05%3A21Z#L984)
> Something is going on in the CI where this function is being called either improperly or the function is not getting declared before it is being called. That's all I know right now.

Do note that the CI does not use Ash's fork ([Shovel](https://shovel.ash258.com/)). Besides, it looks like the checks are successful now.

**Comment by RavenMacDaddy** [comment]
Yep, it was the trailing comma that was pointed out by @BlackPowerade, all good.

**Review by Calinou — APPROVED** [review]
Thanks!

**Pull request merged**